kaldb/src/main/java/com/slack/kaldb/zipkinApi/Service.java

```diff
@@ -0,0 +1,15 @@
+package com.slack.kaldb.zipkinApi;
+
+import java.util.List;
```

Comment:
Looks like this class is not used anymore? Can we remove it? Or if you want to keep as reference just comment it out for now?